app/api/api_v1/endpoints/user.py

```diff
@@ -19,6 +21,37 @@ def register(
     """
     Registration API for the service.
     """
+    if settings.REPORTING_USING_GATEKEEPER:
+        payload = json.dumps(
+            {
+                "username": user_information.email.split("@")[0],
+                "email": user_information.email,
+                "password": user_information.password,
+                "first_name": "Reporting",
+                "last_name": "Backend",
+            }
+        )
+        headers = {"Content-Type": "application/json"}
+        url = settings.REPORTING_GATEKEEPER_BASE_URL + "api/register/"
+
+        try:
+            response = requests.request("POST", url, headers=headers, data=payload)
+            print(response.json())
+            if str(response.status_code)[0] == 2:
+                response = Message(message="You have successfully registered!")
```

Comment:
Maybe a more specific message like `You have successfully registered to Gatekeeper!`